RELEASE_NOTES.md

```diff
@@ -0,0 +1,5 @@
+#### 1.0.1 May 27 2016 ####
+Removed non-generic IPublisher and ISubscriber interfaces.
```

Comment:
Hi Victor,

How would you define feature parity with the jvm version? The API is at feature parity I believe.

I'll make the changes to publish this as a beta release.

Thanks